src/draw/lv_draw_buf.c

```diff
@@ -184,6 +187,8 @@ lv_draw_buf_t * lv_draw_buf_reshape(lv_draw_buf_t * draw_buf, lv_color_format_t
 {
     if(draw_buf == NULL) return NULL;
 
+    /*If color format is invalid, keep using the original format.*/
```

Comment:
Correct text:

> If color format is not defined or unknown, keep using the original color format.

... but I think we could also omit the comment.

Reply:
LVGL never used `cf == 0` , so I tend to comment here to explain the special usage. I'll update the comment. Thank you!